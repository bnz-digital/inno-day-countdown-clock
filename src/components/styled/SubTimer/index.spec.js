import React from 'react'
import { shallow } from 'enzyme'

import StyledSubTimer from '.'

describe('components:styled:SubTimer', () => {
  it('renders <StyledSubTimer/> correctly', () => {
    expect(toJson(shallow(<StyledSubTimer />))).toMatchSnapshot()
  })
})
