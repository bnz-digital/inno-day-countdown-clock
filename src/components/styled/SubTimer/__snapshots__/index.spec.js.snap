// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:SubTimer renders <StyledSubTimer/> correctly 1`] = `
.c0 {
  font-size: 2vw;
}

<p
  className="c0"
/>
`;
